esmvaltool/recipes/recipe_OceanPhysics.yml

```diff
@@ -0,0 +1,502 @@
+      long2:   30.
+      lat1:   -80.
+      lat2:    80.
+    time_average:      
+    regrid:
+      target_grid: 1x1
+      scheme: linear           
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+            
+  prep_map_5: # For a specific region at depth levels,
+    fix_data:
+    fix_metadata:
+    extract_levels:
+      levels:  [0., 10., 100., 1000., ]
+      scheme: linear_extrap
+    time_average:               
+    extract_region:
+      long1:  -80.
+      long2:  30.
+      lat1:   -80.
+      lat2:   80.
+    regrid:
+      target_grid: 1x1
+      scheme: linear            
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean ]
+
+  # --------------------------------------------------
+  # Transects preprocessors
+  # --------------------------------------------------
+  prep_transect_1: # For extracting along a trajectory
+    fix_data:
+    fix_metadata:
+    time_average:
+    extract_trajectory:
+      latitudes: [-5., 5.,]
+      longitudes: [332.,332.]
+      number_points: 4              # VERY SLOW with high numbers!
+     
+  prep_transect_2: # For extracting a transect
+    fix_data:
+    fix_metadata:
+    time_average:
+    extract_transect: # Pacific Equator
+      latitude: 0.
+      longitude: [120., 300.]
+                           
+  prep_transect_3: # For extracting a transect
+    fix_data:
+    fix_metadata:
+    time_average:
+    extract_transect: # Atlantic Meridional Transect
+      latitude: [-50.,50.]  
+      longitude: 332.
+      
+  # --------------------------------------------------
+  # Depth-profile preprocessors
+  # --------------------------------------------------
+  prep_profile_1: # For a global profile
+    fix_data:     
+    fix_metadata: 
+    average_region:
+      coord1: longitude
+      coord2: latitude   
+      
+  prep_profile_2: # For a regional profile
+    fix_data:     
+    fix_metadata: 
+    extract_region:
+      long1: 0.
+      long2:  20.
+      lat1:  -30.
+      lat2:  30.
+    extract_volume:
+      z_min: 0.
+      z_max: 100.
+    average_region:          
+      coord1: longitude
+      coord2: latitude
+                  
+  # --------------------------------------------------
+  # Depth-profile preprocessors
+  # --------------------------------------------------
+  prep_depth_integration_1: # For a 2D map global profile
+    fix_data:     
+    fix_metadata: 
+    depth_integration:
+      coordz: depth
+    time_average:      
+
+       
+     
+diagnostics:
+  # --------------------------------------------------
+  # Depth integrated diagnostics
+  # --------------------------------------------------  
+  diag_depthInt_1:
+    description: Global Ocean Surface mean timeseries
+    variables:       
+      thetao: # Temperature ocean 
+        preprocessor: prep_depth_integration_1      
+        field: TO3M
+    scripts:
+      Global_Ocean_DepthIntegration_map: &Global_Ocean_DepthIntegration_map
```

Comment:
Remove the anchor `&Global_Ocean_DepthIntegration_map`, it looks like you're not using that anywhere. See [here](https://blog.daemonl.com/2016/02/yaml.html) for a short explanation of YAML anchors.